### ISSUE TITLE
Javascript Audio App Meditation APP

### ISSUE THREAD

**Issue by aliya-rahmani** [Vishal-raj-1/Awesome-JavaScript-Projects#438]
Hi ! I am Aliya Rahmani, a GSSOC'21 participant.
I will like to contribute to this project by adding a simple Audio app. It will play some relaxing audio with different themes and simple UI. Made with HTML, CSS, and Javascript.
Please , kindly assign this task to me

**Comment by haarsh01** [comment]
Hi, I am a Gssoc'21 participant, I want to contribute to this issue.
Please assign it to me.
Thank you

**Comment by 991rajat** [comment]
@aliya-rahmani Currently someone is doing this type of project once it is merged,
You could suggest something new can be added or modify to that untill then it's on hold.

**Comment by 991rajat** [comment]
#397 Everyone this type of project is underdevelopment. Once it's developed you can create new issue for modification or new features
I'm closing this issue for now
